buildsystem/modules/FindOgg.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017-2017 the openage authors. See copying.md for legal info.
+# asdf: this file was copied from FindOpus.cmake. check legal stuff.
```

Comment:
this file contains very basic cmake invocations that are common to just finding some library, so i don't think we need to attribute the findopus author.